django/db/models.py

```diff
@@ -226,18 +222,19 @@ class FieldInformation(models.Model):
     manager                 = FieldsManager()
     objects                 = models.Manager() # default manager
     
-    table                   = _CHAR(max_length=35, **_NULLOKSTR)
-    field                   = _CHAR(max_length=35, **_NULLOKSTR)
-    alias                   = _CHAR(max_length=35, **_NULLOKSTR)
-    queryset                = _CHAR(max_length=35, **_NULLOKSTR)
+    table                   = _TEXT(**_NULLOKSTR)
```

Comment:
That's exactly right for Postgres -- that's what I was talking about. But Django has to cater to databases that don't support varchar without a length limit, so it won't let you make a CharField without a max_length.